packages/lib-classifier/src/plugins/tasks/components/InputStatus/InputStatus.js

```diff
@@ -23,27 +24,22 @@ export default function InputStatus ({ count, tool }) {
   }
 
   return (
-    <StyledInputStatus textAlign='end'>
-      <Markdownz>{status}</Markdownz>
+    <StyledInputStatus>
```

Comment:
This status is always strings of count and label provided by the tool models. No need for markdown.